service/project.go

```diff
@@ -566,6 +567,11 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 		uis.LoggedError(w, r, http.StatusInternalServerError, errors.Errorf("expected build baron config but was actually '%T'", i))
 		return
 	}
+	err = bbProjectIsValid(projectRef.Id, buildbaronConfig)
```

Comment:
Not sure if there's currently a way to handle bbProject in the patch route. If there is then we should add validation; if not we should add it (but we could make a new ticket for that).

Reply:
I'll create a ticket for this

Reply:
https://jira.mongodb.org/browse/EVG-15964